src/persist-client/src/cache.rs

```diff
@@ -61,6 +62,10 @@ pub struct PersistClientCache {
     pub(crate) state_cache: Arc<StateCache>,
     pubsub_sender: Arc<dyn PubSubSender>,
     _pubsub_receiver_task: JoinHandle<()>,
+
+    // WIP hack plumb this around instead (What does this mean? Why not have this be `std::sync::OnceLock<Arc<TxnRead>>`?)
```

Comment:
Let me know if you want to brainstorm here!

Reply:
Yes please! I was going to bring this up at the weekly today.